lib/plugin.template.js

```diff
@@ -148,6 +148,22 @@ const setupProgress = (axios, ctx) => {
 }<% } %>
 
 export default (ctx, inject) => {
+  let headers = {
+    common : {
+      'Accept': 'application/json, text/plain, */*'
+    },
+    delete: {},
+    get: {},
+    head: {},
+    post: {},
+    put: {},
+    patch: {}
+  }
+
+  if (process.server) {
+    headers['Accept-Encoding'] = 'gzip, deflate'
```

Comment:
That should be in `common` I think 🤔

Reply:
@manniL @pi0 don't know why but It didn't work for me when it's in common.

tried it locally.